test/test_ops.py

```diff
@@ -943,7 +936,8 @@ def test_bbox_convert_jit(self):
         torch.testing.assert_close(scripted_cxcywh, box_cxcywh, rtol=0.0, atol=TOLERANCE)
 
 
-class BoxAreaTester(unittest.TestCase):
+@cpu_only
+class BoxAreaTester:
```

Comment:
Shouldn't this be 
```suggestion
class TestBoxArea:
```
?